### ISSUE TITLE
index adjustments

### ISSUE THREAD

**Issue by sigu** [lakehub/Aryaprimary_kisumu#15]
1. change copyright to read 2016
2. more content to make it look nice

NB: I have decided to reduce the image size into a container to maintain some aspect ratio
@TheBonface  please look into this

**Comment by AndyCodez** [comment]
@cesswairimu I think we both agree that the "copyright" part is simple.

**Comment by cesswairimu** [comment]
@AndyCodez thats right. @sigu  what do you mean by content?  Is it the design or the words contained in the website

**Comment by sigu** [comment]
@cesswairimu I actually meant the page has less information and looks a bit scattered. More relevant information about the school can be added to the home page. Visitors should see as much information about the school without necessarily going to the next page. If this isnt clear, let me know
